### ISSUE TITLE
some train problem

### ISSUE THREAD

**Issue by dlyldxwl** [chuanqi305/MobileNet-SSD#33]
Thanks for your work, I use kitty dataset and your method  to train a model.lr=0.0005,120000 iter,stepsize=20000,but i have the 48% auccury.can you give some advice?Thanks

**Comment by mna12478** [comment]
@dlyldxwl hi, do you find the reason why the accuracy isn't high? I meet the similar problem.

**Comment by dlyldxwl** [comment]
@mna12478  i add input resolution to 416，and get some of conf layer weight as pretrain model，then acc still less 70，but improve to 65。finally，you can prune tradional conv layer weight  to ensure inference speed

**Comment by mna12478** [comment]
@dlyldxwl thanks for your reply, but I have some questions and needs your help:
1. I follow the step in 'Train your own dataset' in https://github.com/chuanqi305/MobileNet-SSD, should I add some extra operation or should I change some deploy in the deploy files? For example, should I change the train.protxt file? I noticed that someone proposed to delete the '_new' in 'conv11_mbox_conf_new', 'conv13_mbox_conf_new','conv14_2_mbox_conf_new'.
2. When I finetune the model on MSCOCO, the training is very slow, or about 4 second an epoch, did you meet this problem?
3. I test the finetuned model on VOC0712, the result is shown in detect_eval, so what's the difference between the detect_eval and accuracy?

I'm a beginner in object detection.  I would very appreciate if you can help me.
